web/client/new.json

```diff
@@ -51,6 +35,20 @@
         "group": "background",
         "visibility": false
       },
+      {
+        "format": "image/jpeg",
+        "group": "background",
+        "name": "s2cloudless:s2cloudless",
+        "opacity": 1,
+        "title": "Sentinel 2 Cloudless",
+        "type": "wms",
+        "url": [
+          "https://1maps.geo-solutions.it/geoserver/wms", "https://2maps.geo-solutions.it/geoserver/wms", "https://3maps.geo-solutions.it/geoserver/wms", "https://4maps.geo-solutions.it/geoserver/wms", "https://5maps.geo-solutions.it/geoserver/wms", "https://6maps.geo-solutions.it/geoserver/wms"
+        ],
+        "source": "s2cloudless",
```

Comment:
Do we need the source?

Reply:
yes, because of [here](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/components/background/BackgroundSelector.jsx#L62)

Reply:
@MV88, correct, I think we should use type and not source in that place, but this is not related to your PR